buildout-mercator.cfg

```diff
@@ -24,10 +24,10 @@ frontend_package_name = mercator
 backend_package_name = adhocracy_mercator
 
 [test_run_unit]
-package_paths = src/adhocracy_core src/adhocracy_mercator src/adhocracy_sample
+package_paths = src/adhocracy_* src/adhocracy_frontend/adhocracy_frontend/tests/unit
 
```

Comment:
we don not need the line:  ..adhocracy_frontend/tests/unit , theses are only javascript unit tests